client/allocrunner/taskrunner/task_runner.go

```diff
@@ -377,7 +377,8 @@ func NewTaskRunner(config *Config) (*TaskRunner, error) {
 		return nil, err
 	}
 
-	// Initialize the runners hooks.
+	// Initialize the runners hooks. Must come after initDriver so hooks
+	// can use tr.driverCapabilities
```

Comment:
This didn't change. It just took me a while to unravel, so I wanted to document it.